modules/markup/html.go

```diff
@@ -390,35 +385,38 @@ func visitNode(ctx *RenderContext, procs, textProcs []processor, node *html.Node
 		}
 
 		if attr.Key == "class" && attr.Val == "emoji" {
-			textProcs = nil
+			procs = nil
 		}
 	}
 
 	// We ignore code and pre.
 	switch node.Type {
 	case html.TextNode:
-		textNode(ctx, textProcs, node)
+		textNode(ctx, procs, node)
 	case html.ElementNode:
 		if node.Data == "img" {
 			for i, attr := range node.Attr {
 				if attr.Key != "src" {
 					continue
 				}
-				if len(attr.Val) > 0 && !isLinkStr(attr.Val) && !strings.HasPrefix(attr.Val, "data:image/") {
-					prefix := ctx.URLPrefix
+				if len(attr.Val) > 0 && !IsLinkStr(attr.Val) && !strings.HasPrefix(attr.Val, "data:image/") {
+					var base string
 					if ctx.IsWiki {
-						prefix = util.URLJoin(prefix, "wiki", "raw")
+						base = ctx.Links.WikiRawLink()
+					} else if ctx.Links.HasBranchInfo() {
+						base = ctx.Links.MediaLink()
```

Comment:
We mix `/raw/` and `/media/`. Only `/media/` should be used I think to support LFS images.